packages/dso-toolkit/src/components/highlight-box/highlight-box.mixins.scss

```diff
@@ -46,6 +47,18 @@
     margin-top: units.$u2 * -1;
     object-fit: cover;
     width: calc(100% + #{units.$u4});
+
+    @container (min-width: 360px) and (max-width: 470px) {
```

Comment:
Ik heb gekeken of ik hier scss variables in kon gebruiken op een betekenisvolle manier, maar dat is me niet gelukt. Gebruiken van de verwijzingen breken op de een of andere manier de werking van de container queries. Deze units zijn aan de andere kant ook zo specifiek voor de container zelf dat het denk ik niet zoveel toevoegt.

Reply:
Bedoel je dat het niet gelukt is om de waardes uit Sass variables te halen?

En even voor mijn beeldvorming, hoe ben je aan 360px en 470px gekomen?

Reply:
Nee met de syntax gaat er denk ik iets mis in Sass (ook als ik ze probeer te escapen met `#{'%'}`). 

De 360 en 470 komen uit proefondervindelijk het beeld smaller en breder maken om te kijken op welke breedte de afbeelding wel erg hoog is (maar dat is iets wat we met Michiel moeten bekijken, het leek me voor hem makkelijker als er al een voorbeeld stond).

Reply:
Duidelijk.

Moet Michiel nog meekijken voordat dit PR wordt gemerged?

Reply:
Ja, om even te kijken of het gedrag is zoals hij het graag wil hebben.

Reply:
Check! ⏳

Reply:
Michiel komt waarschijnlijk donderdag 26 oktober bij me terug hierover.

Reply:
@MichielDHVD En?

Reply:
Michiel is akkoord!

Reply:
🎉